bee-test/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "bee-test"
+version = "0.1.0"
+authors = [ "IOTA Stiftung" ]
+edition = "2018"
+description = "Test utilities the bee framework"
+readme = "README.md"
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = [ "iota", "tangle", "bee", "framework", "test" ]
+homepage = "https://www.iota.org"
+
+[dependencies]
+rand = { version = "0.8.4", features = [ "min_const_gen" ]}
```

```suggestion
rand = { version = "0.8", features = [ "min_const_gen" ]}
```
Let's omit patches as it gets really annoying to update them all